packages/liferay-jest-junit-reporter/test/index.js

```diff
@@ -53,7 +48,18 @@ const passedTestReport = {
 };
 
 describe('liferay-jest-junit-reporter', () => {
+	let fs;
+	let reporter;
+
 	beforeEach(() => {
+		jest.resetModules();
+
+		jest.mock('fs');
+
+		fs = require('fs');
+
+		reporter = require('../src');
+
```

Comment:
To anybody who drives by this later and wonders about this fix, I could have just moved the `jest.mock()` call down the file — because Jest uses Babel to transparently hoist `jest.mock()` calls up to the top; that would have been a more minimal fix. I will probably follow up and do that in another PR.

Reply:
Follow-up is here: https://github.com/liferay/liferay-npm-tools/pull/299